### PR TITLE
Standardise BASH configurations for Simulator Terminals

### LINKED ISSUE #159: Standardise bash configiurations
Use the same bash configurations for all instances and containers:

* [x] Prompt command / PS1
* [x] inputrc
* [x] aliases
* [x] bash_completion

Take from the attack/launch containers which @sublimino modified already to make more friendly

### PR DESCRIPTION
**Pull request by s-irvine**
This PR standarises the BASH experience for users across terminals and hosts in simulator.

Fixes #159

Redo of #166 

Requires #157 to be merged first!!

### CODE REVIEW

**Comment by raoulmillais** [comment]
Looks good so far :)  Thanks for renaming the "Internal Node" that's been bugging me for a while!

File: launch-files/bashrc [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      . /usr/share/bash-completion/bash_completion
+    elif [ -f /etc/bash_completion ]; then
+      # shellcheck disable=SC1091
+      . /etc/bash_completion
+    fi
+  fi
+}
+
+set_prompt_command() {
+PROMPT_COMMAND=__prompt_command
```
Comment (sublimino) [inline launch-files/bashrc]: run shfmt over this

File: terraform/modules/AWS/Bastion/bashrc [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        . /usr/share/bash-completion/bash_completion
+      elif [ -f /etc/bash_completion ]; then
+        # shellcheck disable=SC1091
+        . /etc/bash_completion
+      fi
+    fi
+  }
+
+  set_prompt_command() {
+  PROMPT_COMMAND=__prompt_command
```
Comment (sublimino) [inline terraform/modules/AWS/Bastion/bashrc]: please run shfmt over this too

File: terraform/modules/AWS/InternalHost/bashrc [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        . /usr/share/bash-completion/bash_completion
+      elif [ -f /etc/bash_completion ]; then
+        # shellcheck disable=SC1091
+        . /etc/bash_completion
+      fi
+    fi
+  }
+
+  set_prompt_command() {
+  PROMPT_COMMAND=__prompt_command
```
Comment (sublimino) [inline terraform/modules/AWS/InternalHost/bashrc]: hehe there's a pattern emerging here

File: terraform/modules/AWS/Kubernetes/bashrc [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        . /usr/share/bash-completion/bash_completion
+      elif [ -f /etc/bash_completion ]; then
+        # shellcheck disable=SC1091
+        . /etc/bash_completion
+      fi
+    fi
+  }
+
+  set_prompt_command() {
+  PROMPT_COMMAND=__prompt_command
```
Comment (sublimino) [inline terraform/modules/AWS/Kubernetes/bashrc]: shfmt

**Comment by s-irvine** [comment]
@sublimino resolved!

**Comment by raoulmillais** [comment]
you are lying @reviewdog!  You cannot put a sheban in a bashrc

**Comment by s-irvine** [comment]
The annoying thing is it has highlighted the line telling shellcheck to ignore that SC :rage:

**Comment by raoulmillais** [comment]
I think the better solution is to use `# shellcheck shell=bash` just in case ignoring that turns off some checks.  It's also a bit more explicit

**Comment by s-irvine** [comment]
That fixed it!

**Pull request merged**